CHANGELOG

```diff
@@ -4,6 +4,11 @@
 - preserve warning functions after call to pytest.deprecated_call. Thanks
   Pieter Mulder for PR.
 
+- fix issue833: --fixtures does not show fixtures defined in test files,
+  except for the first one collected. Thanks Florian Bruhin for reporting
+  and Bruno Oliveira for the PR.
```

Comment:
i find the changelog entry a bit confusing.  it should rather read "--fixtures now shows all fixtures of collected test files, not just the first one" or so.